README.md

```diff
@@ -2,6 +2,11 @@
 
 An open-source handbook of applied guidance and tools for sustainable software development and maintenance.
 
+Please see the following links to access published content:
+
+- __Online (HTML)__: https://software-gardening.github.io/almanac/
+- __Offline (PDF)__: [software-gardening-almanac.pdf](https://software-gardening.github.io/almanac/software-gardening-almanac.pdf)
```

Comment:
Sorry I didn't get to this review in time. Looks good. Only thing I see is this PDF is a broken link. Looks like it's supposed to be generated as part of the build/deploy process.

Reply:
Thanks @vincerubinetti and no worries! This was a forward-facing content addition that relied on a successful deploy after merge (and wasn't available after initially merging as a result of the errors mentioned in #31). Now that the deploy worked, the link should now operate as expected.